_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -62,89 +59,121 @@ sudo su postgres
 psql
 ```
 
-```bash
+```sql
 ALTER USER postgres password 'myStrongPassword';
 ```
 
 Quit psql typing `\q`
 
 Exit postgres user typing `exit`
 
-Navigate to main folder inside postgresql/version/
+Navigate to main folder inside `postgresql/version/`
 ```bash
 cd /etc/postgresql/14/main/
 ```
 We need to edit two files, `pg_hba.conf` and `postgresql.conf`
 ```bash
-sudo nano pg_hba.conf
+sudo nano -w pg_hba.conf
 ```
 Scroll down the file and Add `host, all, all, 0.0.0.0/0, md5`, has to be the first line before `local, all, postgres, , peer`
 
 | TYPE | DATABASE | USER | ADDRESS | METHOD |
 | ---- | -------- | ---- | ------- | ------ |
 | host | all | all | 0.0.0.0/0 | md5 |
 | local | all | postgres |  | peer |
+{: .docs_table}
 
+Close the file and save the changes with `Control+X` and `Y`
 
 ```bash
-sudo nano postgresql.conf
+sudo nano -w postgresql.conf
 ```
 Search for `#listen_addresses='localhost'`, uncomment the line and replace `localhost` for `*`
 
+Close the file and save the changes with `Control+X` and `Y`
+
 Restart the PostgreSQL server
 ```bash
 sudo service postgresql restart
 ```
 
 ### Setup Parse Server
 
-Create a directory
-```bash
-cd ~
-```
+Create a folder called parse-server
 ```bash
-mkdir parse-server
+cd ~ && mkdir parse-server && cd parse-server
 ```
+
+Install Parse Server Locally
 ```bash
-cd parse-server
+sudo yarn add parse-server
 ```
 
-Run the bash script and follow the instructions, the script have some visual issues and the keys generation doesn't work.
-```bash
-sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+Edit the `package.json` file with your preferences, but do not change the start line inside the scripts.
+```jsonc
+{
+  "name": "my-app",
+  "description": "parse-server for my App",
+  "scripts": {
+    "start": "parse-server config.json"
+  },
+  "dependencies": {
+    "parse-server": "^5.2.1"
+  }
+}
+
 ```
-After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+Close the file and save the changes with `Control+X` and `Y`
+
+After that, we need to create a configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+
 ```bash
 sudo nano -w config.json
 ```
-This are the basic options of the config.json file, for the full list you can type `parse-server --help` or refer to the [full options document](https://parseplatform.org/parse-server/api/5.2.0/ParseServerOptions.html) for more details.
+
 ```jsonc
 {
   "appId": "exampleAppId",
   "masterKey": "exampleMasterKey",
-  "clientKey": "exampleClientKey",
   "appName": "MyApp",
-  "cloud": "./cloud/main",
```

Comment:
Why are we removing the cloud code file?

Reply:
Not sure/don't remember right now, probably because I saw somewhere? or because in this guide I didn't explain how to add  the file... ?🤷‍♂️